app/controllers/verify/step_controller.rb

```diff
@@ -8,5 +8,28 @@ class StepController < ApplicationController
     before_action :confirm_idv_attempts_allowed
 
     helper_method :step
+
+    private
+
+    def step_name
+      raise NotImplementedError, 'must implement step_name method'
+    end
```

Comment:
Per this article that Jessie found, can we please stop raising this error? http://chrisstump.online/2016/03/23/stop-abusing-notimplementederror/